MANIFEST.in

```diff
@@ -16,3 +16,8 @@ exclude napari/benchmarks/*
 recursive-exclude resources *
 exclude bundle.py
 exclude dockerfile
+
+# Bundled test package for translations
```

Comment:
do we still need this with the new test fixture?

Reply:
Nope we do not! good catch!